src/UmbracoIdentity/IdentityEnabledMembersMembershipProvider.cs

```diff
@@ -23,5 +53,26 @@ public new bool VerifyPassword(string password, string hashedPassword)
         }
 
         public override MembershipPasswordFormat PasswordFormat => MembershipPasswordFormat.Hashed;
-    }
-}
\ No newline at end of file
+
+        public override void Initialize(string name, NameValueCollection config)
+        {
+            base.Initialize(name, config);
+
+            AllowOnlyAlphanumericUserNames = config.GetValue("allowOnlyAlphanumericUserNames", false);
+            PasswordRequiresDigit = config.GetValue("passwordRequiresDigit", false);
+            PasswordRequiresLowercase = config.GetValue("passwordRequiresLowercase", false);
+            PasswordRequiresUppercase = config.GetValue("passwordRequiresUppercase", false);
+        }
+
+		public override string ToString()
+		{
+			var result = base.ToString();
```

Comment:
Sorry for the delay! What does the base.ToString() return here? Just wondering if we need to override ToString to return debug information? If the base class does this already, then sure

Reply:
Yeah the base class in Core (a few abstracts deep) does this, so it's extending that:

https://github.com/umbraco/Umbraco-CMS/blob/v8/dev/src/Umbraco.Core/Security/MembershipProviderBase.cs#L936

Reply:
ah cool thanks for that. all looks good, i'll get this merged in 🎉